experimental/dds/tree2/src/test/feature-libraries/optional-field/optionalField.spec.ts

```diff
@@ -362,24 +382,27 @@ describe("optionalField", () => {
 				};
 
 				const expected: OptionalChangeset = {
-					fieldChange: {
-						id: brand(1),
-						wasEmpty: true,
-						newContent: {
-							set: { type: brand("value"), value: "X" },
-							buildId: { localId: brand(41) },
-						},
-					},
-					childChanges: [[{ localId: brand(0) }, arbitraryChildChange]],
+					build: [
+						{ id: { localId: brand(41) }, set: { type: brand("value"), value: "X" } },
+					],
+					// TODO:AB#6298: This test case demonstrates a problem with rebasing transactions:
+					// we don't realize that { localId: brand(1) } no longer refers to the right node
+					// because we rebased over a change that detaches that node. We either need to augment
+					// this with a move from { localId: brand(0), revision: taggedBaseChange.revision } => { localId: brand(1) }
```

Comment:
Adding a  move from { localId: brand(0), revision: taggedBaseChange.revision } to { localId: brand(1) } doesn't seem like a viable solution here because { localId: brand(1) } should contain the node inserted by baseChange and removed by the rebased change.